site/assets/js/snippets.js

```diff
@@ -63,11 +63,11 @@
   // Instantiate all toasts in a docs page only
   const toastTrigger = document.getElementById('liveToastBtn')
   const toastLiveExample = document.getElementById('liveToast')
+  const toastBootstrap = new bootstrap.Toast(toastLiveExample)
```

Comment:
Shouldn't we instantiate this only if there's a `toastTrigger`? Doesn't make a lot of sense to do otherwise...